api/aether-top-level-openapi3.yaml

```diff
@@ -134,76 +134,265 @@ components:
     Transaction:
       description: Transaction refers to a multi-target transactional change. Taken from https://github.com/onosproject/onos-api/tree/master/proto/onos/config/v2
       properties:
+        meta:
+          description:
+          properties:
+            key:
+              description:
+              type: string
+            version:
+              description:
```

Comment:
add some simple description e.g. `the version of the Transaction`

Reply:
done